README.md

```diff
@@ -14,33 +14,22 @@
 
 ## <a name="overview"></a>Overview
 
-### Prime Directive
+The Nerdery’s perspective is that code exists to solve a problem. We favor solutions which are pragmatic and innovative over adherence to any one approach, abstraction, or toolset. Because the front-end engineering ecosystem is constantly evolving, a set of guidelines for working within it must continue to be relevant even as specific implementations change. We prioritize interoperability when choosing solutions, and use our guiding principles to write code that is resilient and easy to maintain. We also look to the practices of the front-end community as a whole to create a system of universal consistency.
 
-The standards herein are a guide to keep developers at the Nerdery interoperable amongst each other and able to produce the highest quality code given the timeline and budget constraints of a project.
+This is a living document and is reflective of the opinions and expertise of The Nerdery’s front-end engineering group. However, the discussion of, contribution to, and evolution of this document is reliant on the front-end community both inside and outside of The Nerdery.
 
-For a successful project, always favor solutions that benefit the project and end users over producing "perfect code".
+### Guiding Principles
 
-Always aim to meet the standards knowing you'll have to break the rules at some point. Be thoughtful about when you deviate and check with a senior software engineer or a principal software engineer for guidance.
```

Comment:
Perhaps I'm just getting lost in the language of it then. Ideally, I'd prefer the points you highlighted above were more plainly stated. Something like...

> Solutions should focus on the best approach for a given problem or project. They should not be limited by a strict interpretation of these standards.

> Consider these standards as "best practices"...

My 2¢. Thanks for humoring me. 

Regarding the second part of my point... when a project does need to deviate from the standards, do you think it would be worthwhile documenting why and specifically how any deviations occurred? I'd think this would also help to encourage developers to justify their reasoning as well as provide contextual information for other developers (current and future) on the project.